ports/stm32/boards/OPENMV_N6/mpconfigboard.h

```diff
@@ -0,0 +1,175 @@
+#define MICROPY_HW_BOARD_NAME       "OpenMV N6"
+#define MICROPY_HW_MCU_NAME         "STM32N657X0"
+
+#define MICROPY_GC_STACK_ENTRY_TYPE uint32_t
+#define MICROPY_ALLOC_GC_STACK_SIZE (128)
+#define MICROPY_FATFS_EXFAT         (1)
+
+#define MICROPY_HW_HAS_SWITCH       (0)
+#define MICROPY_HW_HAS_FLASH        (1)
+#define MICROPY_HW_FLASH_MOUNT_AT_BOOT (1)
+#define MICROPY_HW_SDCARD_MOUNT_AT_BOOT (0)
+#define MICROPY_HW_ENABLE_RNG       (1)
+#define MICROPY_HW_ENABLE_RTC       (1)
+#define MICROPY_HW_ENABLE_ADC       (0)
+#define MICROPY_HW_ENABLE_DAC       (0)
+#define MICROPY_HW_ENABLE_USB       (1)
+#define MICROPY_HW_ENABLE_SDCARD    (1)
+#define MICROPY_PY_PYB_LEGACY       (0)
+
+#define MICROPY_HW_ENABLE_INTERNAL_FLASH_STORAGE (0)
+
+#define MICROPY_BOARD_ENTER_BOOTLOADER board_enter_bootloader
+#define MICROPY_BOARD_EARLY_INIT    board_early_init
+#define MICROPY_BOARD_LEAVE_STANDBY board_leave_standby
+
+// HSE is 48MHz, this gives a CPU frequency of 800MHz.
+#define MICROPY_HW_CLK_PLLM         (6)
+#define MICROPY_HW_CLK_PLLN         (100)
+#define MICROPY_HW_CLK_PLLP1        (1)
+#define MICROPY_HW_CLK_PLLP2        (1)
+#define MICROPY_HW_CLK_PLLFRAC      (0)
+
+// The LSE is a 32kHz crystal.
+#define MICROPY_HW_RTC_USE_LSE      (1)
+#define MICROPY_HW_RTC_USE_US       (1)
+
+// External SPI flash.
+#define MICROPY_HW_XSPIFLASH_SIZE_BITS_LOG2 (28) // 256Mbit
+
+// ROMFS config
+#define MICROPY_HW_ROMFS_ENABLE_EXTERNAL_XSPI (1)
+#define MICROPY_HW_ROMFS_XSPI_SPIBDEV_OBJ (&spi_bdev)
+#define MICROPY_HW_ROMFS_ENABLE_PART0 (1)
+
+#if 1
+// SPI flash, block device config.
+#define MICROPY_HW_BDEV_SPIFLASH                (&spi_bdev)
+#define MICROPY_HW_BDEV_SPIFLASH_EXTENDED       (&spi_bdev)
+#define MICROPY_HW_BDEV_SPIFLASH_CONFIG         (&spiflash_config)
+#define MICROPY_HW_BDEV_SPIFLASH_OFFSET_BYTES   (16 * 1024 * 1024)
+#define MICROPY_HW_BDEV_SPIFLASH_SIZE_BYTES     (16 * 1024 * 1024)
```

Comment:
@dpgeorge If I understand correctly, this is the offset at which the flash filesystem (writable fs) starts and its size, correct? If so, this needs to be 4MBs and 4MBs, and the romfs offset in the linker script needs to be at 8MBs and size 24MBs. This is the current flash layout we use:
4MBs -> firmware
4MBs-> RW FS
24MBs-> ROMFS

I can change that, if there's a good reason to put the rw FS last?

Reply:
Seconding Ibrahim here. We've shipped alpha N6 boards to a few customers. Would prefer Micropython matches OpenMV's flash layout so they don't have to update their bootloaders.

Reply:
> I can change that, if there's a good reason to put the rw FS last?

It doesn't matter the order.  I will update this to match your settings.

Reply:
Now updated.